clang/test/CodeGen/debug-dir-win-pe-magic-sections.c

```diff
@@ -0,0 +1,26 @@
+// This test checks if COFF file compiled with
+// -fprofile-generate has magic section ".pgi" to indicate so.
+
+// RUN: %clang --target=x86_64-pc-windows -fprofile-generate %s -c -o %t_x86
```

Comment:
As this test now does compilation for both x86 and aarch64, it also requires that both those targets are enabled for this test to be runnable - so this test at least needs a matching `REQUIRES:` line. 

While it earlier was raised whether this was x86_64 specific, I wonder if it adds much value here to test both, as it (marginally) limits the set of configurations where this test is possible to run overall.

Reply:
There aren't any specific differences between x86 and aarch64 in this change; I am testing both just in case. I see that this potentially limits the set of configuration of testing, but already many COFF tests limit themselves to x86/aarch64, as these are the only platform modern Windows supports

Reply:
Yes - I'm not saying that this should be tested on more platforms - on the contrary.

My point was, consider if we'd have two buildbots, one which only has enabled the x86 target, and another one that only has enabled the aarch64 target - then neither of these will run any of the tests, as they require _both_ x86 and aarch64 to be enabled at the same time. We certainly have lots of build configurations with both targets enabled at the same time though, so it's not a big practical issue though. But if adding the other architecture doesn't add any relevant test coverage, I would just pick one of them (x86 is probably the most often enabled target) and stick with that.

Reply:
Okay, I see your point. I will stick to x86 then